hedera-node/src/test/java/com/hedera/services/txns/schedule/ScheduleCreateTransitionLogicTest.java

```diff
@@ -145,6 +183,252 @@ public void followsHappyPath() {
         verify(txnCtx).setStatus(SUCCESS);
     }
 
+    @Test
+    public void validNotReadyForExecutionOtherPartyKeyNotFound() {
```

Comment:
I think that we should either create new test file called `ScheduleReadyForExecution` and add tests of whats going on inside the `readyForExecutionFunction` or check the logic here. What I mean by that is verify f.e that `store.get`, `ledger.get` and `signingOrder.keysForOtherParties` are called with the correct arguments. 
At the moment, if I delete the whole line that calls the `readyForExecution` the test will not fail which will be pretty strange given that this test should test the `readyForExecution` method